r-package/grf/R/causal_forest.R

```diff
@@ -251,6 +251,14 @@ causal_forest <- function(X, Y, W,
 #'                Xi using only trees that did not use the i-th training example). Note
 #'                that this matrix should have the number of columns as the training
 #'                matrix, and that the columns must appear in the same order.
+#' @param linear.correction.variables Optional subset of indexes for variables to be used in local
+#'                   linear prediction. If NULL, standard GRF prediction is used. Otherwise,
+#'                   we run a locally weighted linear regression on the included variables.
+#'                   Please note that this is a beta feature still in development, and may slow down
+#'                   prediction considerably. Defaults to NULL.
+#' @param ll.lambda Ridge penalty for local linear predictions
+#' @param ll.weight.penalty Option to standardize ridge penalty by covariance (TRUE),
```

Comment:
^ Actually, see longer comment above.

Reply:
Ah this is such a good catch- THANK YOU! The product of 5 million iterations of different versions of this code. I think I've fixed it in the latest commit.

Reply:
@halflearned that is a good point about naming. Maybe we could address this in a follow-up, since we also use the name `ll.weight.penalty` in local linear regression forests.